src/modules/Server/public/Get-SdnVfpVmSwitchPort.ps1

```diff
@@ -5,58 +5,49 @@ function Get-SdnVfpVmSwitchPort {
     <#
```

Comment:
Does it make sense to change `Get-SdnVfpVmSwitchPort` to `Get-SdnVfpPort` to align with command like `Get-SdnVfpPortXX`.?

Reply:
@luyaof I went with `private\Get-VfpVMSwitchPort` and `public\Get-SdnVfpVmSwitchPort` as under the covers, it's using `vfpctrl /list-vmswitch-port` so was hoping that would be clear on data being returned. I am happy to make adjustment though if does not make sense.